int-tests/src/test-integration/java/io/neow3j/compiler/AccountIntegrationTest.java

```diff
@@ -18,26 +21,30 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
+@Testcontainers
```

Comment:
The `@Testcontainers` annotation is obsolete because the container life cycle is managed by the `ContractTestExtension`. This might not be the most elegant way but it's how we implemented it and it's easier to keep it that way. Please remove the annotation on all similar test classes - i.e., everywhere the `ContractTestExtension` is used but not a container as a test class member - and also from `io.neow3j.compiler.ContractTestExtension` itself.

Instead of the extension we could extend a `TestContainer` class and try to pack the `ContractTestExtension` functionality in there. Then the `@Testcontainers` annotation would handle the life cycle. But, I don't think that's necessary at the moment.

Reply:
True, I missed that fact. I'll remove the annotations accordingly.